src/libs/sfdk/sdk.cpp

```diff
@@ -56,6 +57,19 @@ const char VBOXMANAGE_PATH[] = "VBoxManagePath";
 
 Sdk *Sdk::s_instance = nullptr;
 
+const char* separator(Sdk::TextStyle textStyle)
+{
+    switch (textStyle) {
+        case Sdk::TextStyle::CamelCase:
+            return "";
+        case Sdk::TextStyle::SnakeCase:
+            return "_";
+        case Sdk::TextStyle::PrettyText:
+            return "\xC2\xA0";
+    }
+    return "";
+}
+
```

Comment:
- I would prefer moving this to utils_p.h/utils.cpp.
- The enum should be declared in utils.h then (needs to be added).
- If you like use of enum *class* here (I am not decided), then I would prefer using just `TextStyle::Pretty` instead of `TextStyle::PrettyText`.
- I would prefer returning QString here, likely constructed with QStringLiteral

Reply:
Updated